test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "maxParallelThreads": 1,
+  "parallelizeTestCollections": false
+} 
```

Comment:
nit: remove trailing space and add blank line before EOF

Reply:
done